### PR TITLE
fix(firestore-send-email): added logging for invalid email messages, documentation updated with firestore example

### PR DESCRIPTION
**Pull request by dackers86**
fixes: https://github.com/firebase/extensions/issues/599

### CODE REVIEW

File: firestore-send-email/README.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ admin.firestore().collection('mail').add({
 })
 ```
 
+Or in Firestore
+
+| Field       | Type        | Value                |
+| ----------- | ----------- | -----------          |
+| to          | String      | someone@example.com  |
+| message     | Map         | {subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.'} |
```
Comment (Salakar) [inline firestore-send-email/README.md:30]:
```suggestion
| message     | Map         | `{ subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.' }` |
```

File: firestore-send-email/README.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ admin.firestore().collection('mail').add({
 })
 ```
 
+Or in Firestore
+
+| Field       | Type        | Value                |
+| ----------- | ----------- | -----------          |
+| to          | String      | someone@example.com  |
```
Comment (Salakar) [inline firestore-send-email/README.md:29]:
```suggestion
| to          | String      | `someone@example.com`  |
```

File: firestore-send-email/README.md [hunk 3]
```diff
@@ -22,6 +22,14 @@ admin.firestore().collection('mail').add({
 })
 ```
 
+Or in Firestore
+
+| Field       | Type        | Value                |
+| ----------- | ----------- | -----------          |
+| to          | String      | `someone@example.com`  |
+| message     | Map         | `{ subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.' }` |
```
Comment (Salakar) [inline firestore-send-email/README.md:30]:
```suggestion
| to          | String      | `"someone@example.com"`  |
| message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |
```

File: firestore-send-email/README.md [hunk 4]
```diff
@@ -22,6 +22,14 @@ admin.firestore().collection('mail').add({
 })
 ```
 
+Or in Firestore
+
+| Field       | Type        | Value                |
+| ----------- | ----------- | -----------          |
+| to          | String      | `"someone@example.com"`  |
+| message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |
```
Comment (Salakar) [inline firestore-send-email/README.md:30]:
```suggestion
| Field       | Type        | Value                                                                                   |
| ----------- | ----------- | --------------------------------------------------------------------------------------- |
| to          | String      | `"someone@example.com"`                                                                 |
| message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |
```

File: firestore-send-email/README.md [hunk 5]
```diff
@@ -22,6 +22,14 @@ admin.firestore().collection('mail').add({
 })
 ```
 
+Or in Firestore
```
Comment (Salakar) [inline firestore-send-email/README.md:25]:
```suggestion
Or in Firestore the following fields can be added;
```

**Review by russellwheatley — APPROVED** [review]
LGTM

**Comment by huangjeff5** [comment]
Can we delete the .js files from this repo?

**Pull request merged**